facia/public/humans.txt

```diff
@@ -323,3 +323,6 @@ SETI (Tools & Infra): Stephen Geller
 GitHub: stephengeller
 Twitter: _stephengeller
 Location: London, UK
+
+Engineer: Oliver Lloyd
+From: London, UK
```

Comment:
👋 

Minor: you probably want to add your name above line 154: https://github.com/guardian/frontend/blob/104354943b6e9ec88a1ee80f717e1ebd37edaa61/facia/public/humans.txt#L154

Otherwise you are already a former member 😢

Reply:
Ha, thanks, good spot.

Reply:
Ah yep I hadn't noticed this when helping @oliverlloyd !